int8 SET_TASK_GOAL     =  1
int8 SET_TASK_KP       =  2
int8 SET_TASK_KD       =  3
int8 SET_POSTURE_GOAL  =  4
int8 SET_POSTURE_KP    =  5
int8 SET_POSTURE_KD    =  6

int8 GET_TASK_GOAL     =  7
int8 GET_TASK_KP       =  8
int8 GET_TASK_KD       =  9
int8 GET_POSTURE_GOAL  = 10
int8 GET_POSTURE_KP    = 11
int8 GET_POSTURE_KD    = 12

int8 SET_CONTROL_POINT = 13
int8 GET_CONTROL_POINT = 14

int8 mode
float64[] value

---

bool ok
string errstr

float64[] value
float64[] lower_bound
float64[] upper_bound
string[] name
string[] unit
